firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -566,6 +613,58 @@ def test_update_success_200(self):
         self.assert_update_response(data, response_data)
 
 
+class ControlledSiteContentUpdateApiTestMixin:
+    """
+    For use with ControlledBaseSiteContentApiTest
+    """
+
+    @pytest.mark.django_db
+    def test_update_assistant_permissions_valid(self):
+        site, user = factories.get_site_with_member(
+            site_visibility=Visibility.PUBLIC, user_role=Role.ASSISTANT
+        )
+
+        instance = self.create_minimal_instance(site=site, visibility=Visibility.TEAM)
+
+        self.client.force_authenticate(user=user)
+
+        data = self.get_valid_data(site)
+        data["visibility"] = "team"
+
+        response = self.client.put(
+            self.get_detail_endpoint(key=instance.slug, site_slug=site.slug)
+            if instance.__class__ == SitePage
+            else self.get_detail_endpoint(key=instance.id, site_slug=site.slug),
```

Comment:
This could be updated to use the get_lookup_key function. Example here: https://github.com/First-Peoples-Cultural-Council/fv-be/blob/9d9c72d98ce44609147a4e5b080b5ec629295cf9/firstvoices/backend/tests/test_apis/base_api_test.py#L376C33-L376C33

Reply:
Added